src/components/Content/TodoContent.tsx

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterStatus } from '../../helper';
+
+interface Props {
+  todos: Todo[],
+  filter: string,
+  activeTodosQuantity: number,
+  onFilterChange: (arg: FilterStatus) => void,
+}
+export const TodoContent: React.FC<Props> = ({
```

Comment:
i suggest you to create a separate component for `Footer`, `Header`, `TodoList`.